### PR TITLE
Add CGP template and updated the README

### PR DESCRIPTION
**Pull request by aslawson**
- Added CGP template
- Updated README to include CGP instructions
- Updated top level directory to be `celo-proposals/` although it appears I need to rename the github repo separately

### CODE REVIEW

File: CGPs/template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Describe the issue that motivates this CGP. It should indicate all parameters that are being changed and why doing so is important.
+
+Explain what benefits the enhancement this change will bring. To the extent possible, enumerate use cases affected by this CGP.
+
+## Proposed Changes
+
+Fill out the following template for each transaction in the proposal
+
+1. Description of transaction 1
```
Comment (cla-bel) [inline CGPs/template.md:25]: Is the "1" after transaction supposed to be there?

File: CGPs/template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# CIP [0000]: Title for Governance Proposal
+
+- Date: 2020-05-11
+- Author: @githubhandle
```
Comment (cla-bel) [inline CGPs/template.md:4]: Author or Authors?

File: CGPs/template.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- Status: DRAFT|PROPOSED|EXECUTED|EXPIRED
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
```
Comment (cla-bel) [inline CGPs/template.md:7]: "is" or "was" ("has been" generally less  readable)

File: CGPs/template.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	- EXECUTED = The governance proposal was executed on the network
+	- EXPIRED = The governance proposal expired
+- Governance Propsal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+Pick a four digit number subsequent to the latest PR raised and name your file that.
+
+Describe the issue that motivates this CGP. It should indicate all parameters that are being changed and why doing so is important.
```
Comment (cla-bel) [inline CGPs/template.md:17]: Should probably put somewhere on the template what CGP stands for

File: CGPs/template.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
+	- EXECUTED = The governance proposal was executed on the network
+	- EXPIRED = The governance proposal expired
+- Governance Propsal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+Pick a four digit number subsequent to the latest PR raised and name your file that.
```
Comment (cla-bel) [inline CGPs/template.md:15]: This may be hard to understand for non native english speakers. Maybe put an example?

File: CGPs/template.md (hunk 3 above)
Comment (asaj) [reply CGPs/template.md:7]: +1 to "was"

File: CGPs/template.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+# CIP [0000]: Title for Governance Proposal
+
+- Date: 2020-05-11
+- Author: @githubhandle
+- Status: DRAFT|PROPOSED|EXECUTED|EXPIRED
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
+	- EXECUTED = The governance proposal was executed on the network
+	- EXPIRED = The governance proposal expired
+- Governance Propsal ID #: [if submitted]
```
Comment (asaj) [inline CGPs/template.md:10]: typo

File: CGPs/template.md (hunk 1 above)
Comment (asaj) [reply CGPs/template.md:25]: Yup

File: CGPs/template.md (hunk 2 above)
Comment (aslawson) [reply CGPs/template.md:4]: sure will update

File: CGPs/template.md (hunk 5 above)
Comment (aslawson) [reply CGPs/template.md:15]: consistency with the CIP instruction -- ya its kinda confusing for me even.  I'll add example

**Pull request merged**